_includes/cloud-query/cloud-query-open-history.md

```diff
@@ -0,0 +1,6 @@
+* Click the button in the top left labeled **HISTORY**
+
+When the query history is open, you can see list of all of your past queries on the right side of the screen with the most recent queries shown at the top. This list displays the query, the execution time, the rows returned, and when it was run. The color next to each query signifies if the query execution was successful (green), unsuccessful (red), or still running (blue).
```

Comment:
Nope. We don't put an instruction followed by a long explanation.

The rule is:

* Explain first
* Instruct later

So first you need to explain what the Query history is.

Then you tell the user how to activate it